axlearn/common/adapter_torch.py

```diff
@@ -1387,7 +1387,7 @@ def forward(
         )
         # Causal mask.
         return apply_attention_logit_biases(
```

Comment:
Great catch! Could you change `apply_attention_logit_biases` to take `apply_attention_logit_biases` as a keyword arg? This also better matches https://docs.google.com/document/d/1tK3MyfZQgXyrvWNg3rt6NQCItuFnFw_FBd6-C7pE2Wk/edit#heading=h.d1xks5sf41jd.

Feel free to do so in a follow-up PR.

Reply:
sure. added a commit for this. please help review again. thanks.

Reply:
Thanks! Is there any other call site of `apply_attention_logit_biases`?